packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -136,11 +139,11 @@ const BountyDetails = ({ bounty }: { bounty: BountyCollection }): JSX.Element =>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Description</Heading>
-				<Text>{description}</Text>
+				<Text>{ ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }</Text>
```

Comment:
Need to check for description being null or undefined. Some bounties don't have descriptions or criteria, so these will be blank. Crashing:
![image](https://user-images.githubusercontent.com/2557404/167450290-bc710c19-0a14-412f-a04d-646a46c1bd0b.png)